src/main/java/core/basesyntax/generator/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.generator;
+
+import core.basesyntax.Figure;
+import core.basesyntax.figure.Circle;
+import core.basesyntax.figure.IsoscelesTrapezoid;
+import core.basesyntax.figure.Rectangle;
+import core.basesyntax.figure.RightTriangle;
+import core.basesyntax.figure.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 5;
+
+    public static Figure[] generateFigure(int figuresCount) {
+        Figure[] figures = new Figure[figuresCount];
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        for (int i = 0; i < figuresCount; i++) {
+            int figureNumber = random.nextInt(NUMBER_OF_FIGURES);
+            switch (figureNumber) {
+                case 0:
+                    figures[i] = new Circle(random.nextDouble() + random.nextInt(), "circle",
```

Comment:
```suggestion
                    figures[i] = new Circle(random.nextDouble(PASS_SOMETHING_HERE), "circle",
```

same for other nextInt()/nextDouble() calls

Reply:
nextDouble() не приймає жодних значень((((